Specifications/Language/3_Expressions/CallExpressions.md

```diff
@@ -0,0 +1,24 @@
+# Call expressions
+
+Call expressions are an important part of any programming language. Operation and function calls, much like [partial applications](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/3_Expressions/Closures.md#partial-application), can be used as an expression anywhere as long as the returned value is of a suitable type.
+
+The usefulness of calling functions in this form primarily lies in debugging, and such operation calls are one of the most common constructs in any Q# program. At the same time, operations can only be called from within other operations and not from within functions. For more information, see also [Qubits](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/4_TypeSystem/QuantumDataTypes.md#qubits).
```

Comment:
"The usefulness of calling functions" sounds weird when the second half of the sentence says "operation calls" since we generally either draw a distinction between functions and operations or use a more generic term. Maybe something like "The usefulness of call expressions" just to stay consistent with the title?

Reply:
Oh, I see the context now, the sentence realy is about operations. Maybe the first part should say "operation" instead of "function" or "call expressions using operations" or something similar.

Reply:
I've changed "function" to "operation" per your suggestions.